build_tools/packaging/tests/promote_from_rc_to_final_test.py

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/env python
+
+import argparse
+import shutil
+import sys
+import os
+from pathlib import Path
+import tempfile
+from packaging.version import Version
+from pkginfo import Wheel
+import subprocess
+import urllib
+
+sys.path.insert(0, os.fspath(Path(__file__).parent.parent))
+import promote_from_rc_to_final
+
+
+def checkPromotedFileNames(dir_path: Path, platform: str) -> tuple[bool, str]:
+    if platform == "linux":
+        expected_promoted_pkgs = [
+            "rocm-7.9.0.tar.gz",
+            "rocm_sdk_core-7.9.0-py3-none-linux_x86_64.whl",
+            "rocm_sdk_devel-7.9.0-py3-none-linux_x86_64.whl",
+            "rocm_sdk_libraries_gfx94x_dcgpu-7.9.0-py3-none-linux_x86_64.whl",
+            "pytorch_triton_rocm-3.3.1+rocm7.9.0-cp312-cp312-linux_x86_64.whl",
+            "torch-2.7.1+rocm7.9.0-cp312-cp312-linux_x86_64.whl",
+            "torchaudio-2.7.1a0+rocm7.9.0-cp312-cp312-linux_x86_64.whl",
+            "torchvision-0.22.1+rocm7.9.0-cp312-cp312-linux_x86_64.whl",
+        ]
+    else:
+        expected_promoted_pkgs = [
+            "rocm-7.9.0.tar.gz",
+            "rocm_sdk_core-7.9.0-py3-none-win_amd64.whl",
+            "rocm_sdk_devel-7.9.0-py3-none-win_amd64.whl",
+            "rocm_sdk_libraries_gfx1151-7.9.0-py3-none-win_amd64.whl",
+            "torch-2.9.0+rocm7.9.0-cp312-cp312-win_amd64.whl",
+            "torchaudio-2.9.0+rocm7.9.0-cp312-cp312-win_amd64.whl",
+            "torchvision-0.24.0+rocm7.9.0-cp312-cp312-win_amd64.whl",
+            "pytorch_triton_rocm-3.3.1+rocm7.9.0-cp312-cp312-linux_x86_64.whl",
```

Comment:
no triton on Windows
```suggestion
            "torchvision-0.24.0+rocm7.9.0-cp312-cp312-win_amd64.whl",
```